src/Components/Reports/Util.js

```diff
@@ -1,6 +1,13 @@
 import { get } from 'lodash';
+import moment from 'moment';
 import { pdfRowKeys, percentageKeys, reportRowKeys, SYSTEMS_REPORT_FILE_NAME } from '../../constants';
 
+export const formatLastReported = (date) => {
+    const today = moment().format('YYYY-MM-DD');
+    const isTodayWithoutTime =  today === date;
+    return isTodayWithoutTime ? 'Today' : moment(date).fromNow();
```

Comment:
If a system is updated/uploaded today then it's showing `Today` not as `Just now` or `X minutes ago`.

![Screenshot from 2022-04-11 20-19-46](https://user-images.githubusercontent.com/61837065/162766658-f30760ca-6bd0-4c31-b0eb-c0a1cfbd4887.png)

Reply:
Hello @r14chandra that's expected as we have only date information coming from the backend for `report_date` field, once the backend will send time information as well this code will handle that. Please refer to below test cases for same.

Reply:
Hello @r14chandra that's expected as we have only date information coming from the backend for `report_date` field, once the backend will send time information as well this code will handle that. Please refer to below test cases for same.

Reply:
Got it.